### ISSUE TITLE
Multi load for performance test

### ISSUE THREAD

**Issue by nirmalchandra** [authorjapps/zerocode#88]
This is probably very useful feature to test the `stress` or `capacity` of an application under multiple type of load/stress e.g.
1. One user is just firing a request `payload` with one record to create one entity on server
1. Another user firing a request `payload` with 10 records to create multiple entities on server
1. Another user is firing similar POST, then PUT and then GET to check CRUD operation went well
1. Another user just firing GET to view the results from server due it has only read-only access

I want to generate load or stress with the above scenarios. 

Basically if I configure `4 threads(mean 4 users)` with 4 secs ramp-up time, I want `all 4 threads` to be picked up in parallel for all `4 scenarios` respectively.
 
That means if I configure 8 threads, then I want to repeat the same requests to be fired again in circular manner till all my threads are used up i.e. `thread-5 firing case-1, thread-6 firing case-2, thread-7 firing case-3, thread-8 firing case-4` and so on.

Which means if I configure 10 threads, then `thread-9 firing case-1, thread-10 firing case-2` again and so on in similar pattern.

Can I use `ZeroCodeLoadRunner.class` for this?
If so how?

**Comment by authorjapps** [comment]
@nirmalchandra , you need a `MultiLoadRunner` to achieve this. Watch this space for details. Thanks

**Comment by authorjapps** [comment]
This(below) should solve the purpose-
(Browse this file in the [example repo](https://github.com/authorjapps/performance-tests/blob/master/src/test/java/org/jsmart/zerocode/samples/load/parallelmulti/LoadMultipleGetPostPutTest.java))
```
@LoadWith("load_generation.properties")
@TestMapping(testClass = GetScreeningServiceTest.class, testMethod = "testGetScreeningLocalAndGlobal")
@TestMapping(testClass = PostCorpLoanServiceTest.class, testMethod = "testPostNewLoan_crudOperations")
@TestMapping(testClass = PutCorpLoanServiceTest.class, testMethod = "testPutAmendExistingLoan")
@RunWith(ZeroCodeMultiLoadRunner.class)
public class LoadMultipleGetPostPutTest {
}
```

**Comment by authorjapps** [comment]
Fixed and released in the version [1.2.1](https://github.com/authorjapps/zerocode/releases/tag/zerocode-rest-bdd-1.2.1).

Usage examples and helps are  [here](https://github.com/authorjapps/performance-tests#performance-testing-testing-load-and-stress).
